content/usgs-packages/Application_Intro.Rmd

```diff
@@ -32,14 +32,16 @@ set.seed(1)
 
 ## Lesson Summary
 
-TBD
+This lesson will revisit the packages taught in previous lessons and use some of their functions to create a common data analysis workflow in a modular and reproducible manner. Then, you can learn how a modular approach compares to traditional analyses.
```

Comment:
I like this idea - I think it would help people who are not familiar understand the concept of modularity a bit better.

Not sure what you mean by the "promise" from the second sentence?